drafts/one-percent-for-open-source.rst

```diff
@@ -0,0 +1,111 @@
+.. post::
+   :tags: python, opensource, sustainability, sustain
+
+One Percent for Open Source
+===========================
+
+The open source ecosystem is the `most valuable part`_ of the software industry today.
+All of the companies and technologies that we build today are enabled by open source software.
+From the programming languages to the web frameworks,
+the operating system to the cryptography,
+all of it is open source.
+
+Open source has a `funding problem`_.
+There are many `warning signs`_ that open source is an ecosystem in danger.
+
+Open source is maintained for "free" by people all over the world,
+enabled by the internet (which is also built on open source).
+Many people doing this work are burning out,
+giving up,
+or just never standing in the first place.
+We need to improve the sustainability of open source in order to protect the valuable ecosystem that we have built,
+and help it grow in the future.
+
+Funding Open Source
+-------------------
+
+I'd like to propose an idea:
+
+    A program called **1% for open source**: A program where Venture Capitalist's and startup companies agree to invest 1% of their funding round into the open source software that they use. 
+
+I believe that this is a win/win/win for all involved:
+
+* **Venture Capitalists** invest in the open source ecosystem that makes companies *much cheaper to build*. It's also *really good marketing* for the VC firms, to show they are investing in the common tools we all rely on.
+* **Startups** invest in the tools that they use, to *ensure continued support and maintenance*, to remove risk from their future, and have better tools to build their company with. It will also make *hiring developers* much easier for the companies, because developers love companies that invest in open source.
```

Comment:
Citation needed. 😄 

srsly - It's easy to make a blanket statement like this, but is there any actual evidence that it's true, outside of our little bubble of maintainers who want stuff maintained?

Reply:
Might be clearer to split out the benefits to startups? a) hiring b) risk reduction